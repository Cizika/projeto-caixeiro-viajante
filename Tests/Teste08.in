8
1 2 100
1 3 50
1 4 150
1 5 115
1 6 120
1 7 90
1 8 110
2 3 100
2 4 115
2 5 80
2 6 120
2 7 90
2 8 150
3 4 70
3 5 90
3 6 140
3 7 115
3 8 65
4 5 100
4 6 80
4 7 140
4 8 115
5 6 100
5 7 80
5 8 95
6 7 100
6 8 120
7 8 100