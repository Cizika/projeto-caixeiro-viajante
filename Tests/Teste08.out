
0
1
1-2-5-7-6-4-8-3-1
670
